test/udev-test.pl

```diff
@@ -1554,6 +1554,17 @@ sub udev_setup {
                 return 0;
         }
 
+        # check if we are permitted to create block device nodes
+        my $block_device_filename = $udev_dev . "/sda";
+        if (system("mknod", $block_device_filename, "b", "8", "0")) {
+                warn "unable to create $block_device_filename";
+                return 0;
+        }
+        else
+        {
```

Comment:
please follow coding style, we usually place `} else {` on the same line.

Reply:
OK. I've removed the else block as advised below.